lib/checkCriticalPullRequest.js

```diff
@@ -0,0 +1,137 @@
+const {
+  getAllChangedFiles,
+  hasCriticalLabel,
+  CRITICAL_LABEL,
+} = require('./checkPullRequestJob');
```

Comment:
We should add this in a utility module and then use in both the libs.

Reply:
Done.